File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.cpp
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        unordered_map<int,int> freq;
        int maxFreq=0;
        int result=0;

        for(int i=0;i<nums.size();i++){

            freq[nums[i]]++;  

            if(freq[nums[i]]>maxFreq){
                maxFreq=freq[nums[i]];
                result=maxFreq;
            }else if(maxFreq == freq[nums[i]]){
                result+=maxFreq;
            }
        }

        return result;
    }
};

File: 3242-count-elements-with-maximum-frequency/3242-count-elements-with-maximum-frequency.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxFrequencyElements = function(nums) {
    
    const map=new Map();

   for (let i = 0; i < nums.length; i++) {
  if (map.has(nums[i])) {
    map.set(nums[i], map.get(nums[i]) + 1);
  } else {
    map.set(nums[i], 1);
  }
}


    const maxFreq=Math.max(...map.values())

    let maxCount=0;
    for(const [key,value] of map ){
        if(value==maxFreq){
        maxCount++;
        }
    }

    return maxCount * maxFreq;
};